cmd/eksctl-anywhere/cmd/cleanup.go

```diff
@@ -9,7 +9,7 @@ import (
 	"github.com/aws/eks-anywhere/pkg/types"
 )
 
-func cleanup(ctx context.Context, deps *dependencies.Dependencies, commandErr error) {
+func cleanup(ctx context.Context, deps *dependencies.Dependencies, commandErr *error) {
 	close(ctx, deps)
 
 	if commandErr == nil {
```

Comment:
```suggestion
	if *commandErr == nil {
```

Reply:
https://go.dev/play/p/kDa2v0oDGMf

played around with that a bit, looks like we do need a pointer there

Reply:
fixed